earth_enterprise/src/fusion/autoingest/AssetFactory.h

```diff
@@ -304,13 +327,13 @@ namespace AssetFactory
       return asset->MyUpdate(needed);
   }
 
-  template <class MutableDerivedVersionHandleType, class ConfigType, class VersionDType>
+  template <class MutableDerivedVersionHandleType, class ConfigType, class VersionDType, class CachedInputType>
   MutableDerivedVersionHandleType ReuseOrMakeAndUpdate(const std::string& ref_,
                                                        AssetDefs::Type type_,
                                                        const std::vector<SharedString>& inputs_,
                                                        const khMetaData& meta_,
                                                        const ConfigType& config_,
-                                                      const std::vector<AssetVersion>& cachedinputs_)
+                                                      const std::vector<CachedInputType>& cachedinputs_)
```

Comment:
Minor, and not your change, but this line doesn't have the same indentation as the others. Would be nice to fix it.